inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/sink/SinkResourceListener.java

```diff
@@ -67,7 +68,22 @@ public ListenerResult listen(WorkflowContext context) {
         GroupResourceProcessForm form = (GroupResourceProcessForm) context.getProcessForm();
         String groupId = form.getInlongGroupId();
         log.info("begin to create sink resources for groupId={}", groupId);
-        groupService.updateStatus(groupId, GroupStatus.CONFIG_ING.getCode(), context.getOperator());
+        GroupOperateType operateType = form.getGroupOperateType();
+        String operator = context.getOperator();
+        switch (operateType) {
```

Comment:
After updating the StreamSink, it should only change the status of InlongStream.

Reply:
In addition, StreamSink does not have operations such as init, suspend and restart, so these operations should not be judged here.

Reply:
done